Corzbank.Api/Controllers/UsersController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Corzbank.Data.Entities.Models;
+using Corzbank.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace Corzbank.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class UsersController: ControllerBase
+    {
+        private readonly IUserService _userService;
+        public UsersController(IUserService userService)
+        {
+            _userService = userService;
+        }
+
+        [HttpGet]
+        public IActionResult GetUsers()
+        {
+            var result = _userService.GetUsers();
+
+            return Ok(result.Result);
+        }
+
+        [HttpGet("{id}")]
+        public IActionResult GetUserById(Guid id)
```

Comment:
Guid? Is that default type for Identity Id?

If so, change Id in KeyEntity model so everywhere we will have same ID types

Reply:
resolved